subprojects/build-init/src/test/groovy/org/gradle/buildinit/plugins/internal/maven/MavenProjectsCreatorSpec.groovy

```diff
@@ -113,4 +113,31 @@ class MavenProjectsCreatorSpec extends Specification {
         def ex = thrown(MavenConversionException)
         ex.message == "Unable to create Maven project model. The POM file $pom does not exist."
     }
+
+    def "creates projects with compileOnly library"() {
+        given:
+            def pom = temp.file("pom.xml")
+            pom.text = """<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>util</groupId>
+  <artifactId>util</artifactId>
+  <version>2.5</version>
+  <packaging>jar</packaging>
+  <dependencies>
+    <dependency>
+        <groupId>org.gradle</groupId>
+        <artifactId>build-init</artifactId>
+        <version>1.0.0</version>
+        <scope>provided</scope>
+    </dependency>
+  </dependencies>
+</project>"""
+        def mavenProjects = creator.create(settings.buildSettings(), pom)
+        def converter = new Maven2Gradle(mavenProjects, temp.root.getCanonicalFile())
+
+        when:
```

Comment:
Can you align the statements below `when` and `then` as being use in other test methods? For better readability can you also add a new line between `when` and `then`?